src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1545,12 +1545,42 @@ void RaftPart::processAppendLogRequest(const cpp2::AppendLogRequest& req,
       // previously in fact. There are two choise: ask leader to send logs after committedLogId_ or
       // just do nothing.
       if (req.get_last_log_id_sent() < committedLogId_ ||
-          wal_->lastLogId() < req.get_last_log_id_sent() ||
-          wal_->getLogTerm(req.get_last_log_id_sent()) != req.get_last_log_term_sent()) {
+          wal_->lastLogId() < req.get_last_log_id_sent()) {
+        // case 1 and case 2
```

Comment:
so `case 1` is last_log_id < committedLogId_ and `case 2` is wal_->lastLogId() < get_last_log_id_sent ?

Reply:
Yeah, see the comment above in line 1539